doc/samples.md

```diff
@@ -102,5 +103,60 @@ To use the Azure DevOps Extension in a hosted agent using a Release Pipeline, ex
 6. Add the script - either via file or inline. For the example, the script has been included inline. 
 ![PowerShell](/doc/images/Script.PNG)
 
-The first line of the script installs the Azure CLI and adds the Azure DevOps Extension. 
+Including the inline script for reference
+```bash
+pip install --pre azure-cli==2.0.55 --extra-index-url https://azurecliprod.blob.core.windows.net/edgeaz --h
+az --version
+az extension add --name azure-devops
+az devops -h
+```
+
+The first line of the script installs the Azure CLI. 
+
+# Use the Azure DevOps Extension with YAML
+If you prefer to use YAML to provide your release pipeline configuration, you can use the following example to understand how YAML can be used to install Azure CLI and add the Azure DevOps extension.
+
+In the example, you will learn how to add the Azure DevOps extension to Azure CLI and run the build and PR list commands on Linux and Mac OS hosted agent pools.
+
+1. Create the azure-pipelines-steps.yml file and include the content below.
+```
+steps:
+- script: az --version
+  displayName: 'Show Azure CLI version'
+
+- script: az extension add -n azure-devops
+  displayName: 'Install Azure DevOps Extension'
+
+- script: echo ${AZURE_DEVOPS_CLI_PAT} | az devops login
+  env:
+    AZURE_DEVOPS_CLI_PAT: $(System.AccessToken)

```

Comment:
one problem I see with this script is that it is targeting the same org/ project where it is running, can we add example on how to pass these variables when targeting other accounts? 
because I think that will be a more prominent scenario to start with

Reply:
Replace $(System.AccessToken) with the PAT? Won't that make the PAT available in plain text?

Reply:
I meant we can put this in variable in release/ build/ pipeline and use that instead of using system accesstoken

Reply:
Isn't that more details to explain? We need to explain setting up variables and then using it. Further, I am assuming the 80% case of using this would be go to a project -> create a new pipeline or from GitHub -> add app and create a new pipeline. In both, the system tokens should work.